test/e2e/redundancy/gateway_failover.go

```diff
@@ -100,6 +132,13 @@ func testGatewayFailOverScenario(f *subFramework.Framework) {
 	By(fmt.Sprintf("Setting the gateway label for node %q to true", initialNonGatewayNode.Name))
 	f.SetGatewayLabelOnNode(framework.ClusterA, initialNonGatewayNode.Name, true)
 
+	By(fmt.Sprintf("Ensuring that two Gateways become available in cluster %q", clusterAName))
+
+	gwActive := f.AwaitGatewayWithStatus(framework.ClusterA, initialGatewayNode.Name, subv1.HAStatusActive)
+	gwPassive := f.AwaitGatewayWithStatus(framework.ClusterA, initialNonGatewayNode.Name, subv1.HAStatusPassive)
+	Expect(gwActive.Status.Connections).ToNot(BeEmpty(), "The active gateway must have active connections")
```

Comment:
As above, it would be safest to call `AwaitGatewayFullyConnected` for the active gateway.

Reply:
True.

Reply:
Done.